package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Ceciil-service",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.jsx",
```

Comment:
I think the `main: "index.jsx"` line is causing your server to crash (refer to screenshot). I tried running your server locally and it looks like nodemon tried to run both the server/index.js and the index.jsx files. Not sure exactly what happened, but we can sort it out tomorrow. Everything else seems to be running fine, good work!
<img width="723" alt="screen shot 2019-02-07 at 8 37 04 pm" src="https://user-images.githubusercontent.com/25331378/52459160-59f5c800-2b18-11e9-8428-e214f8173914.png">

Reply:
I added a server file so that `npm run server` can start.